notebooks/Quickstart/quickstart.ipynb

```diff
@@ -0,0 +1,259 @@
+    "                                                        url='https://cromwell.caas-prod.broadinstitute.org')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Authenticate with Cromwell with no Auth"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# NotImplemented"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5. [Recommended] Authenticate with Cromwell using the standardized method"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cromwell_tools.cromwell_auth import CromwellAuth\n",
+    "\n",
+    "auth = CromwellAuth.harmonize_credentials(service_account_key='path/to/service/account/key.json', \n",
+    "                                          url='https://cromwell.caas-prod.broadinstitute.org')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submit jobs to Cromwell/Cromwell-as-a-Service"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cromwell_tools import api\n",
+    "from cromwell_tools import utilities"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "response = api.submit(auth=auth_httpbasic, \n",
+    "                      wdl_file='Examples/hello_world.wdl', \n",
+    "                      inputs_files=['Examples/inputs.json'], \n",
+    "                      dependencies=['Examples/helloworld.wdl'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'id': '8375999d-c30e-4dc8-bb79-39ee7835c793', 'status': 'Submitted'}"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response.json()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "All workflows succeeded!\n"
+     ]
+    }
+   ],
+   "source": [
+    "api.wait(workflow_ids=[response.json()['id']], auth=auth_httpbasic)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Query jobs in Cromwell/Cromwell-as-a-Service"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cromwell_tools import api\n",
+    "from cromwell_tools import utilities"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "custom_query_dict = {\n",
```

Comment:
For this example, how about querying for the hello world wdl by it's name or id?

Reply:
Sure!